make/migrations/postgresql/0040_2.1.0_schema.up.sql

```diff
@@ -111,4 +111,6 @@ BEGIN
 END $$;
 
 ALTER TABLE schedule DROP COLUMN IF EXISTS job_id;
-ALTER TABLE schedule DROP COLUMN IF EXISTS status;
\ No newline at end of file
+ALTER TABLE schedule DROP COLUMN IF EXISTS status;
+
+ALTER TABLE artifact ADD COLUMN icon varchar(64);
```

Comment:
64 is enought to store a digtest string?

Reply:
ok.